defe/formatter.py

```diff
@@ -27,8 +28,24 @@ def defy_prompt(feed):
                     input(Fore.GREEN + Style.BRIGHT + "Enter Feed Index to open : ")
                 )
                 print(Style.RESET_ALL)
-                print(Style.BRIGHT + "Opening Link in your default browser ...")
-                webbrowser.open(feed[int(feed_to_open) - 1].link)
+                if podcasts:
+                	player = vlc.MediaPlayer(feed[int(feed_to_open) -1]["links"][0]["href"])
+                	player.play()
+					#VLC module throws errors very often while using; the try-except block catches them
```

Comment:
It doesn't seem to catch the usual steam error output

![pr-output](https://user-images.githubusercontent.com/34342551/89705096-fef49980-d977-11ea-8c57-112d2857a701.png)

See [this thread](https://stackoverflow.com/questions/55897360/how-to-get-vlc-to-shut-up-with-its-prefetch-errors) for a workaround to see if we can hide these ouput